core/controllers/questions_list.py

```diff
@@ -0,0 +1,64 @@
+# Copyright 2019 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Controllers for the questions list in topic editors and skill editors."""
+
+from constants import constants
+from core.controllers import acl_decorators
+from core.controllers import base
+from core.domain import question_services
+from core.domain import skill_domain
+from core.domain import skill_services
+import feconf
+
+
+class QuestionsListHandler(base.BaseHandler):
+    """Manages receiving of all question summaries for display in topic editor
+    and skill editor page.
+    """
+    GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
+
+    @acl_decorators.open_access
+    def get(self, skill_ids):
+        """Handles GET requests."""
+        start_cursor = self.request.get('cursor')
```

Comment:
Do we need to do any validation here?

Reply:
I'm not sure about what you meant by validation. Did you mean validating the cursor? I added validation for splitting the skill_ids, if that is what you mean.

Reply:
Hmm but what should I validate about the cursor? I was looking at other similar code in the codebase but I didn't see any validation about it.

Reply:
I'm not sure -- it was a legitimate question. I was unsure.